packages/junokit/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "junokit",
-  "version": "0.0.5",
-  "main": "dist/index.js",
+  "version": "0.0.6",
+  "main": "dist/src/index.js",
```

Comment:
Please restore this. I optimized the build.

Reply:
Restore how?

Reply:
`"main": "dist/src/index.js",` -> `"main": "dist/index.js",`